Assets/Scripts/Managers/CustomerSpawnManager.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CustomerSpawnManager : MonoBehaviour
+{
+    [SerializeField] private GameObject[] customersList;
+    [SerializeField] private CustomerConfiguration customerConfiguration;
+
+    private void Start()
+    {
+        InvokeRepeating(nameof(SpawnCustomer), customerConfiguration.SpawnFrequencyTime, customerConfiguration.SpawnFrequencyTime);    
+    }
+
+    private void SpawnCustomer()
+    {
+        int attempts = 0; // Avoid infinty loop
+
+        while (attempts < customersList.Length)
```

Comment:
Can we use a for loop instead of while? Can iterate `customersList` until the first inactive one is found. No need to pick random ones.